sources/corepackUtils.ts

```diff
@@ -260,6 +264,30 @@ export async function installVersion(installTarget: string, locator: Locator, {s
   };
 }
 
+async function renameUnderWindows(oldPath: fs.PathLike, newPath: fs.PathLike) {
+  // Windows malicious file analysis blocks files after download so we need to wait for file release
```

Comment:
Does it know that it was downloaded or is it any new file?

Reply:
It depends on what the installed antivirus software decides to check
I will change "files after download" to "files currently under analysis"